packages/opencensus-core/src/stats/view.ts

```diff
@@ -221,18 +230,40 @@ export class BaseView implements View {
    */
   private toPoint(timestamp: Timestamp, data: AggregationData): Point {
     let value;
-
     if (data.type === AggregationType.DISTRIBUTION) {
-      // TODO: Add examplar transition
-      const {count, sum, sumOfSquaredDeviation} = data;
+      const {count, sum, sumOfSquaredDeviation, exemplars} = data;
+      const buckets = [];
+      for (let bucket = 0; bucket < data.bucketCounts.length; bucket++) {
+        const bucketCount = data.bucketCounts[bucket];
+        let statsExemplar;
```

Comment:
Rather than a `let`, what would you think about using `const` and a ternary expression, so:
`const statsExemplar = exmplars ? exemplars[bucket] : undefined;`?

Reply:
Accepted.